sdb/commands/spl/avl.py

```diff
@@ -27,6 +27,7 @@ class Avl(sdb.Walker):
 
     names = ["avl"]
     input_type = "avl_tree_t *"
+    load_on = [sdb.Module("zfs")]
```

Comment:
strictly speaking all of the commands within the `spl` repo are part of the `spl` module, bug given that the `spl` module is not really used for anything else, and the `zfs` module cannot really work without `zfs` I'm fine with this left as-is